src/github.com/getlantern/flashlight/genconfig/cloud.yaml.tmpl

```diff
@@ -1,7 +1,7 @@
 # cloud.yaml contains the default configuration that's made available on the
 # internet.
 client:
-  servers:
+  frontedservers:
```

Comment:
So this will break compatibility with old clients right? Is the plan to just version the file name again?